session-5/task3.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+func main() {
+
+	x := 10
+	y := 20
+
+	println("Before swapping :", x, y)
+	swap(&x, &y)
+	println("After swapping:", x, y)
+
+}
+
+func swap(a *int, b *int) {
+	x := *a
+	*a = *b
+	*b = x
```

Comment:
Elave pointer istifade etmeden inline ptr variable ile hell etmek daha seliqeli olar
```suggestion
*a, *b = *b, *a
```